his_checker.py

```diff
@@ -9,9 +9,44 @@
 
 import sys
 import argparse
+import os
+
+COMMENTS_THRESHOLD = 0.2
+
+def process_comments(files):
+
+    """Process comment density within a file"""
+
+    metric_fail = 0
+    statements_tool = "pmccabe -c "
+    pmccabe_stat_index = 2
+    total_statements = 0
+    comments_tool = "pygount --format=cloc-xml "
+    comment_index = 21
+    comment_density = 0
+    license_comment_size = 4
+    print("Checking the file comment density")
+    for file in files.files:
+        sline = os.popen(statements_tool + str(file)).read().split('\n')
+        for fields in [line.split('\t') for line in sline[:-1]]:
+            statements = int(fields[pmccabe_stat_index])
+            total_statements = total_statements + statements
```

Comment:
If we are iterating file by file, we only need the last line of the pmccabe tool to parse, right? So, we know the index of the `Statements` cell that we want to read. Maybe I'm missing something, but I don't understand why are we iterating each field or why we have a total of statements as we were counting different files.